src/index.mako.html

```diff
@@ -50,24 +50,24 @@
     <meta property="og:title" content="Swiss Geoportal"/>
     <meta property="og:description" content="geo.admin.ch ist die Geoinformationsplattform der Schweizerischen Eidgenossenschaft. // geo.admin.ch est la plateforme de géoinformation de la Confédération suisse."/>
     <meta property="og:type" content="WebApplication"/>
-    <meta property="og:url" content="${application_url}"/>
-    <meta property="og:image" content="${application_url}/${versionslashed}img/logo_geoportal.png"/>
+    <meta property="og:url" content="//map.geo.admin.ch"/>
+    <meta property="og:image" content="//cms.geo.admin.ch/img/logo_geoportal.png"/>
```

Comment:
Please use full URL's here. I think the cards have troubles without fully qualified URLS. Use https. Same as below.